OMPython/__init__.py

```diff
@@ -238,8 +239,11 @@ def sendExpression(self, command):
             self._omc = None
             return result
           else:
-            answer = OMTypedParser.parseString(result)
-            return answer
+            try:
+              answer = OMTypedParser.parseString(result)
+              return answer
+            except:
```

Comment:
I agree that we should catch PyPArsingExceptions instead of all exceptions and should print its message.

Returning `result` is just fine. Basically from this function we want to return the result of the OMC API call done via `sendExpression`. `OMTypedParser.parseString` is used to format the result in a proper python type instead of just returning string type. So if `parseString` is successful return the formatted result i.e., `answer` otherwise just return the `result`.